study1/week2/LongestPalindrome.java

```diff
@@ -0,0 +1,28 @@
+class LongestPalindrome {
+
+    public int longestPalindrome(String s) {
+        int n = s.length();
+        int odd = 0;
+        HashMap<Character, Integer> list = new HashMap<>();
```

Comment:
the `list` variable seems to not reflect what it's actually doing, and seems to be less descriptive. 
something like `countChar` should describe what it's doing.